google/ggrc-core
# GGRC-1785: Unify titles of fields for Tasks and Cycle Tasks

# Issue description

Unify titles of fields for Tasks and Cycle Tasks on New Task, Edit Tasks, Task Info pane, Task export/import, Global and Advanced Search.

# Steps to test the changes

1.Log into GGRC app(login as Admin).
2. Goto Export page.
3. Select the Object Type ( Task Group Task or Cycle Tasks) and verify the displayed names and export the csv file.
5. Open the excel.
6. Verify the attribute names.
7. Open Task info-pane/Cycle task info-pane and check for the displayed field names.
8. Click on Edit for tasks  and check if Task description is there for Rich text/ Checkbox types.
9.Edit some fields, then check for the Change log page and mentioned names there.
10. Check the Global search fields.
11. Check the Advanced search fields.
12. Tast the order by cliecking arrows beside the field name.
# Solution description

Changes are done acrosse multiple pages, to fix the issues mentioned in the ticket.

# Sanity checklist




# PR Review checklist
